src/main/java/school/lemon/changerequest/java/introduction/pr1/HammingDistance.java

```diff
@@ -1,7 +1,23 @@
 package school.lemon.changerequest.java.introduction.pr1;
 
+import java.util.Scanner;
+
 public class HammingDistance {
     public static int compute(String a, String a1) {
-        return -1;
+
+        int lengthA = a.length();
+        int lengthA1 = a1.length();
+        int number = 0; //Different
+        if (lengthA == lengthA1) {
+            for (int i = 0; i < lengthA; i++) {
+                if (a.charAt(i) != a1.charAt(i)) {
+                    number++;
+                }
+            }
+        } else {
+            return -1;
+
+        }
+        return number;
```

Comment:
You can check arguments length at the very first line of function - so you won'y need additional `else` statement.


also `number` is not very good name for hamming distance - the best will be `hammingDistance`.